src/main/java/com/powsybl/openloadflow/dc/equations/ClosedBranchSide1DcFlowEquationTerm.java

```diff
@@ -38,18 +38,18 @@ public static ClosedBranchSide1DcFlowEquationTerm create(LfBranch branch, LfBus
     @Override
     protected double calculateSensi(double ph1, double ph2, double a1) {
         double deltaPhase = ph2 - ph1 + A2 - a1;
-        return -power * deltaPhase;
+        return -calculatePower(element.getPiModel()) * deltaPhase;
     }
```

Comment:
If you had a proper notifiation mechanism you wouldn't have to recalculate the value each time and could continue to cache it.

Reply:
Fixed in commit [c03d566](https://github.com/powsybl/powsybl-open-loadflow/commit/c03d5665c12b1526ebcd99b35cfde5acacb2a123).